exposed/src/main/kotlin/org/jetbrains/exposed/sql/vendors/Mysql.kt

```diff
@@ -69,9 +69,12 @@ internal open class MysqlFunctionProvider : FunctionProvider() {
 open class MysqlDialect : VendorDialect(dialectName, MysqlDataTypeProvider, MysqlFunctionProvider.INSTANSE) {
 
     override fun isAllowedAsColumnDefault(e: Expression<*>): Boolean {
-        val expression = e.toQueryBuilder(QueryBuilder(false)).toString().trim()
-        return super.isAllowedAsColumnDefault(e) ||
-                (expression == "CURRENT_TIMESTAMP" && TransactionManager.current().db.isVersionCovers(BigDecimal("5.6")))
+        val queryBuilder = QueryBuilder(false)
+        e.toQueryBuilder(queryBuilder)
+        val expression = queryBuilder.toString().trim()
```

Comment:
This code could be replaced with e.toString().trim() as `Expression.toString() = QueryBuilder(false).append(this).toString()`

And here you loose early return in a case when `super.isAllowedAsColumnDefault(e)` is `true`

Reply:
All of the issues are done.